build.rs

```diff
@@ -30,6 +30,11 @@ fn snapshot_from_fs_path(path: &Path) -> io::Result<VfsSnapshot> {
                 continue;
             }
 
+            // Ignore images in msgpack-luau because they aren't UTF-8 encoded.
+            if file_name.ends_with(".png") {
+                continue;
+            }
+
```

Comment:
This could probably stand to be made more robust rather than special-casing png files, but I'm also willing to push that off into a different PR since this is primarily plugin-facing.